libs/libqmi/Makefile

```diff
@@ -0,0 +1,78 @@
+#
+# Copyright (C) 2016 Velocloud Inc.
+# Copyright (C) 2016 Aleksander Morgado <aleksander@aleksander.es>
+#
+# This is free software, licensed under the GNU General Public License v2.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libqmi
+PKG_VERSION:=1.22.4
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=git://anongit.freedesktop.org/libqmi
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=f192348840fa3beeb2c65f2423d90476cffe5767
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_SOURCE_VERSION).tar.gz
+
+PKG_MAINTAINER:=Nicholas Smith <nicholas.smith@telcoantennas.com.au>
+
+PKG_FIXUP:=autoreconf
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/nls.mk
+
+define Package/libqmi
+  SECTION:=libs
+  CATEGORY:=Libraries
+  DEPENDS:=+glib2 +libmbim +kmod-usb-net +kmod-usb-net-qmi-wwan +kmod-usb-wdm
```

Comment:
Can be simplified to just +libmbim

Reply:
Really? why not depend on all the actual dependencies? the kmod packages are not required during build, but the system won't be able to use libqmi unless those packages are installed.

Reply:
libmbim includes all of theee

Reply:
if libmbim includes +kmod-usb-net-qmi-wwan as dep that would be an error.

Anyway, I think it is much clearer if libqmi doesn't rely on whatever deps its own deps have. libqmi should build-depend on its direct deps, regardless of whether some of those deps already depend on other deps. Won't happen, but what if libmbim ends up not depending on glib2 on a future version?